src/linker/Linker/MessageOrigin.cs

```diff
@@ -19,6 +22,23 @@ public MessageOrigin (string fileName, int sourceLine = 0, int sourceColumn = 0)
 			SourceColumn = sourceColumn;
 		}
 
+		public static MessageOrigin? TryGetOrigin (IMemberDefinition sourceMethod, int reflectionCallOffset)
```

Comment:
Should be `ilOffset` instead of `reflectionCallOffset`. Similarly `reflectionCall` should have a better name as there is nothing reflection specific on the sequence point here